test/user.js

```diff
@@ -456,3 +456,30 @@ describe('token expiry', () => {
     });
   });
 });
+
+describe('/users/sessions/invalidate_all POST', () => {
+  it('should fail because user logged out', done => {
+    factory.create('user').then(user => {
+      const tk = token.encode({ email: user.email });
+      chai
+        .request(server)
+        .post('/users/sessions/invalidate_all')
+        .set(token.headerName, tk)
```

Comment:
Get several tokens from app (login), check that they are valid and later invalidate.